### ISSUE TITLE
macos m1 arm64 version - Not using the GPU 

### ISSUE THREAD

**Issue by realisticattorney** [Anjok07/ultimatevocalremovergui#754]
Hey guys. Just to notify it's clearly using the CPU cores even though I picked a model with GPU conversion?

<img width="1227" alt="Screenshot 2023-08-21 at 2 36 29 PM" src="https://github.com/Anjok07/ultimatevocalremovergui/assets/66647706/9a1a4199-7bf6-472c-9259-ad25746d4699">

**Comment by ceebu** [comment]
AFAIK, UVR supports only NVidia GPU.

**Comment by an-internet-user001** [comment]
i've spent alot of time researching this as i have a MacStudio with the M2.   Apple does not expose its GPU's for acceleration through NVIDIA.  It must be done through their own ML framework or with "Metal."  I think Tensorflow and Pytorch allow for this, but to reiterate if it comes via NVIDIA it wont work.

**Comment by XReyRobert** [comment]
fun fact: UVR (VR Architecture) models use radeon GPU on intel macos version. I would guess through mps...

**Comment by an-internet-user001** [comment]
Yeah, i guess i should qualify my statement a bit more and say on M1/M2 Macs.   On intel based, whole other story i would assume.
